react/ListPriceRange.tsx

```diff
@@ -75,7 +78,13 @@ function ListPriceRange({
 
   const commercialOffer = seller?.commertialOffer
 
-  if (!commercialOffer || commercialOffer?.AvailableQuantity <= 0) {
+  if (!commercialOffer) {
+    return null
+  }
+
+  const { AvailableQuantity } = commercialOffer
+
+  if (hideWhenUnavailable({ showWhenUnavailable, AvailableQuantity })) {
```

Comment:
For readability, I always prefer to keep one logic inside the if statement when possible. I think it's easier for other developers to pick up the idea I am trying to convey. That said, I will follow your suggestion, since you are more familiar with the guidelines in this code base.

Reply:
@filafb you are right. I suggested to use predicates because they have a meaning and, also, avoid to repeat code. What do you think about it?

@victorhmp @icazevedo @lariciamota can you left your suggestion?

Reply:
Hey @igorbrasileiro. I'm good with the suggestion. Already in the last version I pushed. Thanks!